rules/the-best-xamarin-resources/rule.md

```diff
@@ -1,55 +1,55 @@
 ---
 type: rule
-archivedreason: 
-title: Do you know where to find the best Xamarin resources?
-guid: b30559d3-4656-4cda-b1a2-fc1637b435c7
+title: Do you know where to find the best .NET MAUI resources?
 uri: the-best-xamarin-resources
-created: 2020-10-08T23:13:23.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
-- title: Matt Goldman
-  url: https://ssw.com.au/people/matt-goldman
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Matt Goldman
+    url: https://ssw.com.au/people/matt-goldman
 related: []
 redirects:
-- do-you-know-where-to-find-the-best-xamarin-resources
-
+  - do-you-know-where-to-find-the-best-xamarin-resources
+created: 2020-10-08T23:13:23.000Z
+archivedreason: null
+guid: b30559d3-4656-4cda-b1a2-fc1637b435c7
 ---
 
-The following is a list of resources for learning, staying up to date with, and using Xamarin.
+The following is a list of resources for learning, staying up to date with, and using .NET MAUI.
 
 <!--endintro-->
 
 ### General
 
-**[Awesome Xamarin Forms](https://github.com/jsuarezruiz/awesome-xamarin-forms)** is a curated list on GitHub of a bunch of resources for Xamarin. This includes controls, effects, behaviors, tools, and more. It's a good 'one-stop-shop' for everything Xamarin. 
+**[Awesome .NET MAUI](https://github.com/jsuarezruiz/awesome-dotnet-maui)** (evolution of [Awesome Xamarin Forms](https://github.com/jsuarezruiz/awesome-xamarin-forms)) is a curated list on GitHub of a bunch of resources for .NET MAUI. This includes controls, effects, behaviors, tools, and more. It's a good 'one-stop-shop' for everything related to .NET MAUI. 
 
-**[Weekly Xamarin](http://weeklyxamarin.com/)** is a weekly newsletter sent every Friday or Saturday. It's a community-curated list of all the latest Xamarin news, blog posts, and podcasts.
+**[Weekly Xamarin](http://weeklyxamarin.com/)** is a weekly newsletter sent every Friday or Saturday. It's a community-curated list of all the latest .NET MAUI and Xamarin news, blog posts, and podcasts.
```

Comment:
This isn't around anymore, remove this and add Sands of MAUI

Reply:
Fixed